PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/common/SparkSubmitModel.java

```diff
@@ -287,6 +287,16 @@ public synchronized void setTableModel(@NotNull SubmissionTableModel tableModel)
         return Arrays.stream(SparkSubmissionParameter.defaultParameters);
     }
 
+    @Attribute("sparkevents_relativepath")
+    public String getSparkEventsRelativePath() {
```

Comment:
Please remove `getSparkEventsRelativePath` and `setSparkEventsRelativePath` method in `SparkSubmitModel`. These two methods should be in sub-class `CosmosServerlessSparkSubmitModel`